src/Stratis.CirrusPegD/Program.cs

```diff
@@ -113,16 +105,15 @@ private static IFullNode GetSidechainFullNode(string[] args)
                 MinProtocolVersion = ProtocolVersion.ALT_PROTOCOL_VERSION
             };
 
-            bool enableFedKicking = nodeSettings.ConfigReader.GetOrDefault("enablefedkicking", true);
-            ((PoAConsensusOptions)nodeSettings.Network.Consensus.Options).AutoKickIdleMembers = enableFedKicking;
-
             IFullNode node = new FullNodeBuilder()
                 .UseNodeSettings(nodeSettings)
                 .UseBlockStore()
-                .SetCounterChainNetwork(MainChainNetworks[nodeSettings.Network.NetworkType]())
-                .UseFederatedPegPoAMining()
+                .SetCounterChainNetwork(StraxNetwork.MainChainNetworks[nodeSettings.Network.NetworkType]())
+                .AddPoAFeature()
+                .UsePoAConsensus()
```

Comment:
Can these be combined?

Reply:
Good point, I will add this and that other change as well to a following PR.

Reply:
Ah I see now that we can't as 2 fluent methods add different features to the node (PoaFeature and ConsenusFeature)